config/credentials.yml.enc

```diff
@@ -0,0 +1,2 @@
+7nEADfp0Cjg2FNH0rkiOZhvAS4B2r36/DBUDtVUwvgdojeTEWy4HPJAMPPzKi/G1JK0mS5+KRIwonSSOKMXdrZ+GT6qA81DWYzgIkV8NWJbWsfBpByPo2ZfHY+bN8M+1N3IzTQjPsNCKG7C5qMudLi2kcdBD+QPmXu9zLwqhLYpLAum6aVROtn8K+17d0ZOPodyS8dgO6YYldU4Cy/o21hdrJtADU0xBwsZKnfw9sDqrnlLuunwWtqPM8N8vsbD+rLizJjBmmsteCxtpBsnSyfuFmFJfEu3sI/ewRPi8lzH6zhWSTnxQZFDEztfmedfp9VYDz3H2rDOJMpp/9sbFl4+YA+M4wArmrKAuM5YHbS65Qr02vLRWBh+ivTZo5YdFqo3hI6tL1JkZjWX/JxNGkkK0xfF6--wgyx6nCyHBcXgUAg--Vf6Y8nKp1Bf7pq9VGB7uCg==
```

Comment:
purpose of this file?

Reply:
Bhai, `credentials.yml.enc `is to securely store sensitive information in an encrypted format. This way, the information is protected from unauthorized access, and only those with the decryption key can view or use it.